src/main/webapp/js/projects/react/components/ProjectPreview.js

```diff
@@ -28,7 +34,7 @@ export default function ProjectPreview(props) {
             <strong className="h5 text-success mr-2">
               {projectPreview.numMentors}
             </strong>
-            Mentors
+              Mentors
```

Comment:
```suggestion
            Mentors
```
Do we need to indent this more? It's still within the `span`, right?

Reply:
Agreed, will make the change now.